### DISCUSSION TITLE
api/graphql": x509: certificate is valid for *.numericable.fr, numericable.fr, not [my user]

### DISCUSSION THREAD

**Discussion by selouti** [community/community#77321] [Repositories]
### Select Topic Area

Bug

### Body

I was able to repo but today I keep getting x509 on any repository i try. Tried on Ubuntu and Windows / Win-Cyg machines.

✓ Authentication complete.
- gh config set -h github.com git_protocol https
✓ Configured git protocol
✓ Logged in as selouti
tim@tim-ideacentre-A340-22AST:/opt$ gh repo clone selouti/activemq-artemis:2.19
Post "https://selouti/api/graphql": x509: certificate is valid for *.numericable.fr, numericable.fr, not selouti
tim@tim-ideacentre-A340-22AST:/opt$ gh repo clone apache/activemq-artemis:2.19
Post "https://apache/api/graphql": dial tcp 81.220.255.20:443: i/o timeout

**Comment by JeremJR** [comment]
Hello,
I've found your message while searching for information regarding a similar issue (for others sites).
I assume that you are impacted by the infrastructure of your internet provider, see this http://shaarli.guiguishow.info/?NZPEPw
On my side I'm trying others dns servers as explained in this page.
